Lib/test/test_pty.py

```diff
@@ -324,6 +319,69 @@ def test_master_read(self):
         os.close(master_fd)
         self.assertEqual(data, b"")
 
+    @expectedFailureIfStdinIsTTYAndHAVE_WINCH
+    def test_winch(self):
```

Comment:
New test! This demonstrates that `pty.spawn()` is currently incapable of dynamically resizing the slave end of the pty based on changes in size of the current stdin. See before and after pictures here: https://bugs.python.org/file49455/before.png, https://bugs.python.org/file49456/after.png